sources/packages/web/src/components/generic/BodyHeader.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="mb-4">
+    <span class="category-header-large color-blue">
+      {{ fullTitleMessage }}
+    </span>
+    <div class="float-right">
```

Comment:
in summary screens. we do have a search box too right? so, are we planning to create a slot for the search box or it will be inside the `button` slot?

Reply:
Good question. If the search is present frequently if can have it even here.
For now the idea was just create something to start. Maybe we can rename it to a more generic name. What do you think?

Reply:
Slot renamed to _actions_, as per our conversation 😉